vignettes/wrapping-apis.Rmd

```diff
@@ -538,6 +568,8 @@ resp %>% resp_body_json()
 That body looks pretty useful!
 So let's write a function that extracts the data and formats it for presentation to the user:
 
+JENNY COMMENT HERE
```

Comment:
It feels like all these functions should probably be prefixed with `gist_` instead of `github_`.

Also, kind of like rate limiting, I have yet to work with an API that structures its errors consistently enough to really do this. That would of course be great! I wonder how typical this 🤔 Of course, I know I will be able to take my extremely ugly version of this for Google errors and insert them into this framework. So what I'm really wondering is what these error-body-processing functions will really look like in the wild.

Reply:
Agreed.

Yeah, and in my experience typically there's no way to find out how each endpoint actually structures its errors without deliberately forcing them. I'll add a note to that effect.

Reply:
And, with most APIs, it can be very hard to intentionally cause errors and capture them for study. You basically have to wait for yourself or a user to experience it and then respond.